2-Authorization-I/1-call-graph/AppCreationScripts/sample.json

```diff
@@ -0,0 +1,55 @@
+{
+    "Sample": {
+        "Title": "React single-page application using MSAL React to call the Microsoft Graph API",
+        "Level": 100,
+        "Client": "React SPA",
+        "RepositoryUrl": "ms-identity-javascript-react-tutorial",
+        "Endpoint": "AAD v2.0"
+    },
+    "AADApps": [
+        {
+            "Id": "spa",
+            "Name": "ms-identity-react-c2s1",
+            "Kind": "SinglePageApplication",
+            "HomePage": "http://localhost:3000/",
+            "ReplyUrls": "http://localhost:3000/",
+            "Audience": "AzureADMyOrg",
+            "ManualSteps": [
+                {
+                    "Comment": "Navigate to the Manifest page, find the 'replyUrlsWithType' section and change the type of redirect URI to 'Spa'"
+                }
+            ],
+            "RequiredResourcesAccess": [
+                {
+                  "Resource": "Microsoft Graph",
+                  "DelegatedPermissions": [ "User.Read" ]
+                },
+                {
+                    "Resource": "Microsoft Graph",
+                    "DelegatedPermissions": [ "Mail.Read" ]
+                },
```

Comment:
Remove this as this is failing the configure

Reply:
done!